plugins/ci/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 1.7.0
+## 2.0.0
```

Comment:
1.7.0 should be fine.

Reply:
I think @rbren asked for 2.0.0 because this may impact some pipelines